VideoApi/VideoApi.Services/AudioPlatformService.cs

```diff
@@ -79,28 +79,30 @@ public async Task<WowzaMonitorStreamResponse> GetAudioStreamMonitoringInfoAsync(
 
         public async Task<WowzaGetStreamRecorderResponse> GetAudioStreamInfoAsync(string application, string recorder)
         {
+            const string errorMessageTemplate = "Failed to get the Wowza stream recorder for: {recorder}, StatusCode: {ex.StatusCode}, Error: {ex.Message}";
             var responses = new List<HttpResponseMessage>();
-            try
+            foreach (var client in _wowzaClients)
             {
-                foreach (var client in _wowzaClients)
-                    responses.Add(await client.GetStreamRecorderAsync(application, _configuration.ServerName, _configuration.HostName, recorder));
-                responses.Remove(null);
-                
-                if (responses.All(e => !e.IsSuccessStatusCode)) 
+                try
                 {
-                    var errorMessage = await responses.First().Content.ReadAsStringAsync();
-                    throw new AudioPlatformException(errorMessage, responses.First().StatusCode);
+                    var response = await client.GetStreamRecorderAsync(application, _configuration.ServerName, _configuration.HostName, recorder);
+
+                    if (response.IsSuccessStatusCode)
+                    {
+                        _logger.LogInformation("Got Wowza stream recorder for application: {recorder}", recorder);
+                        return JsonConvert.DeserializeObject<WowzaGetStreamRecorderResponse>(await response.Content.ReadAsStringAsync());
+                    }
+                    responses.Add(response);
+                }
+                catch(Exception ex)
+                {
+                    responses.Add(new HttpResponseMessage(HttpStatusCode.InternalServerError){Content = new StringContent(ex.Message)});
                 }
-                var successfulResponse = responses.First(e => e.IsSuccessStatusCode);
-                _logger.LogInformation("Got Wowza stream recorder for application: {recorder}", recorder);
-                return JsonConvert.DeserializeObject<WowzaGetStreamRecorderResponse>(await successfulResponse.Content.ReadAsStringAsync());
-            }
-            catch (AudioPlatformException ex)
-            {
-                var errorMessageTemplate = "Failed to get the Wowza stream recorder for: {recorder}, StatusCode: {ex.StatusCode}, Error: {ex.Message}";                
-                LogError(ex, errorMessageTemplate, recorder, ex.StatusCode, ex.Message);
-                return null;
             }
+            var errorMessage = await responses.First().Content.ReadAsStringAsync();
```

Comment:
I think an aggregate exception would be better so then you can display all the exceptions, incase the error is different